nautilus_core/backtest/src/matching_engine/tests.rs

```diff
@@ -896,6 +901,7 @@ fn test_matching_engine_valid_market_buy(
 #[rstest]
 fn test_process_limit_post_only_order_that_would_be_a_taker(
     instrument_eth_usdt: InstrumentAny,
+    orderbook_delta_sell: OrderBookDelta,
```

Comment:
It might be better to have a helper (or `OrderBookDeltaBuilder`) for creating the delta which would make the price of 1500 clearer, which is important for the behavior of the test (and we'll want to be able to change price and size of deltas often in tests).

I think this fixture style could be reserved for fairly static things, without special values which are important to the test (such as `cache`, etc)?